src/uct/ib/rc/base/rc_iface.h

```diff
@@ -18,10 +18,12 @@
 #include <ucs/debug/log.h>
 
 
-#define UCT_RC_QP_TABLE_ORDER       12
-#define UCT_RC_QP_TABLE_SIZE        UCS_BIT(UCT_RC_QP_TABLE_ORDER)
-#define UCT_RC_QP_TABLE_MEMB_ORDER  (UCT_IB_QPN_ORDER - UCT_RC_QP_TABLE_ORDER)
-#define UCT_RC_QP_MAX_RETRY_COUNT   7
+#define UCT_RC_QP_TABLE_ORDER            12
+#define UCT_RC_QP_TABLE_SIZE             UCS_BIT(UCT_RC_QP_TABLE_ORDER)
+#define UCT_RC_QP_TABLE_MEMB_ORDER       (UCT_IB_QPN_ORDER - \
+                                          UCT_RC_QP_TABLE_ORDER)
+#define UCT_RC_QP_MAX_RETRY_COUNT        7
+#define UCT_RC_IFACE_FLUSH_REMOTE_LENGTH 8
```

Comment:
I think it should be defined in ib_md.h and influence the sizes of md->flush_mr and md->flush_dvmr

Reply:
> I think it should be defined in ib_md.h and influence the sizes of md->flush_mr and md->flush_dvmr

done